schema/query/predicate_parser.go

```diff
@@ -247,7 +247,11 @@ func (p *predicateParser) parseCommand(field string) (Expression, error) {
 			if !p.expect('}') {
 				return nil, fmt.Errorf("%s: expected '}' got %q", label, p.peek())
 			}
-			return &Regex{Field: field, Value: re}, nil
+			negated := false
+			if label == opNot {
+				negated = true
+			}
```

Comment:
can be written without a branch condition
```suggestion
			negated := label == opNot 
```

Reply:
maybe fix this as well (squash it in, then I can merge).

Reply:
done